kong/tools/database_cache.lua

```diff
@@ -2,12 +2,19 @@ local utils = require "kong.tools.utils"
 local resty_lock = require "resty.lock"
 local json_encode = require("cjson.safe").encode
 local json_decode = require("cjson.safe").decode
+local lrucache = require "resty.lrucache"
```

Comment:
what do we expect the average cache hit ratio to look like? if we expect a lot of cache misses and purges, should we consider `resty.lrucache.pureffi` (or making it configurable?)